mkosi/distributions/debian.py

```diff
@@ -159,19 +159,20 @@ def architecture(arch: Architecture) -> str:
 
 
 def setup_apt(state: MkosiState, repos: Sequence[str]) -> None:
-    state.workspace.joinpath("apt").mkdir(exist_ok=True)
-    state.workspace.joinpath("apt/apt.conf.d").mkdir(exist_ok=True)
-    state.workspace.joinpath("apt/preferences.d").mkdir(exist_ok=True)
-    state.workspace.joinpath("apt/sources.list.d").mkdir(exist_ok=True)
-    state.workspace.joinpath("apt/log").mkdir(exist_ok=True)
+    state.workspace.joinpath("pkgmngr/etc/apt").mkdir(exist_ok=True, parents=True)
```

Comment:
Let's add a helper method to `MkosiState` to access this directory like we have a helper for the staging directory..